[INFO ] Logging to: /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/outdir_train_configuration/RandomForest_Iris_scenario/log-run5.txt
[INFO ] Version of SMAC is v2.10.03-master-778 (3ee628ef9bf2), running on OpenJDK 64-Bit Server VM (1.8.0_442) and Linux 4.18.0-553.36.1.el8_10.x86_64 (amd64) 
[INFO ] Call String: smac --scenario-file Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/RandomForest_Iris_scenario.txt --seed 5 
[INFO ] SMAC started at: Apr 11, 2025 1:27:18 PM. Minimizing mean quality.
[INFO ] First incumbent: config 1 (internal ID: 0x000B), with mean quality: -0.9333333333333333; estimate based on 1 runs.
[INFO ] Sample call for new incumbent config 1 (internal ID: 0x000B): 
cd /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example; /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 5219272 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '2' -min_weight_fraction_leaf '0.0' -n_estimators '100' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:38_33412_712100081.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:38_33412_712100081.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:38_33412_712100081.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.028878172754747244", "min_samples_leaf": "17", "min_samples_split": "70", "min_weight_fraction_leaf": "0.23625363112487868", "n_estimators": "440", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "5219272", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 5219272 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.028878172754747244' -min_samples_leaf '17' -min_samples_split '70' -min_weight_fraction_leaf '0.23625363112487868' -n_estimators '440'  
[WARN ] Detected CRASHED run. The Solution Quality of CRASHED runs will be transformed to the MAX(quality, 1.0E9), to disable this use --transform-crashed-quality false to change the value use --transform-crashed-quality-value
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:53_33615_987867143.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:53_33615_987867143.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:53_33615_987867143.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "entropy", "max_depth_type": "None", "max_features_float": "0.9914373307591141", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples_type": "None", "min_impurity_decrease": "0.0899122498705276", "min_samples_leaf": "86", "min_samples_split": "29", "min_weight_fraction_leaf": "0.2957512654057705", "n_estimators": "68", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "5219272", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 5219272 -bootstrap 'True' -criterion 'entropy' -max_depth_type 'None' -max_features_float '0.9914373307591141' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0899122498705276' -min_samples_leaf '86' -min_samples_split '29' -min_weight_fraction_leaf '0.2957512654057705' -n_estimators '68' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:39_34377_605322502.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:39_34377_605322502.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:39_34377_605322502.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "gini", "max_depth": "4", "max_depth_type": "int", "max_features_float": "0.48941333432252987", "max_features_type": "float", "max_leaf_nodes": "936", "max_leaf_nodes_type": "int", "max_samples": "0.6802160144853284", "max_samples_type": "float", "min_impurity_decrease": "0.006293799662795974", "min_samples_leaf": "60", "min_samples_split": "56", "min_weight_fraction_leaf": "0.012723590393703355", "n_estimators": "228", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "8623161", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 8623161 -bootstrap 'True' -criterion 'gini' -max_depth '4' -max_depth_type 'int' -max_features_float '0.48941333432252987' -max_features_type 'float' -max_leaf_nodes '936' -max_leaf_nodes_type 'int' -max_samples '0.6802160144853284' -max_samples_type 'float' -min_impurity_decrease '0.006293799662795974' -min_samples_leaf '60' -min_samples_split '56' -min_weight_fraction_leaf '0.012723590393703355' -n_estimators '228' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:54_34564_2835912000.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:54_34564_2835912000.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:54_34564_2835912000.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth_type": "None", "max_features_special": "None", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.4868181020266249", "min_samples_leaf": "12", "min_samples_split": "30", "min_weight_fraction_leaf": "0.1083782668141196", "n_estimators": "444", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "8623161", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 8623161 -bootstrap 'False' -criterion 'log_loss' -max_depth_type 'None' -max_features_special 'None' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.4868181020266249' -min_samples_leaf '12' -min_samples_split '30' -min_weight_fraction_leaf '0.1083782668141196' -n_estimators '444'  
